.github/workflows/release.yml

```diff
@@ -0,0 +1,83 @@
+name: Build and upload SQLite3 WASM
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  compile_sqlite3_wasm:
+    strategy:
+      matrix:
+        os: [macos-latest]
+
+    name: Compile sqlite3 wasm for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Setup Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+      - name: Install Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          flutter-version: "3.x"
+          channel: "stable"
+      - name: Setup macOS
+        if: runner.os == 'macOS'
```

Comment:
is this ever not running on macos?

Reply:
Removed those references, wanted to set it up for ubuntu but there isn't a need for it.